lib/resources/package.rb

```diff
@@ -98,10 +98,22 @@ def info(package_name)
       assignment_re: /^\s*([^:]*?)\s*:\s*(.*?)\s*$/,
       multiple_values: false,
     ).params
+    # On some (all?) systems, the linebreak before the vendor line is missing
+    if params['Version'] =~ /\s*Vendor:/
+      v = params['Version'].split(' ')[0]
```

Comment:
Can you provide an example where you get this? I would like to add this in our integration tests. I need to double-check if we can fix the problem before we start parsing the data. This would allow use just `params['Version']` and `params['Release']`